targets/unity-v2/source/Shared/Public/PlayFabApiSettings.cs.ejs

```diff
@@ -10,7 +10,7 @@ namespace PlayFab
         };
 
         /// <summary> This is only for customers running a private cluster.  Generally you shouldn't touch this </summary>
-        public string ProductionEnvironmentUrl = ".playfabapi.com";
+        public string ProductionEnvironmentUrl = "playfabapi.com";
```

Comment:
public string ProductionEnvironmentUrl = PlayFabSettings.ProductionEnvironmentUrl;
I don't want that same string hard coded in many places.

Reply:
Ok let me try that. Thanks!!

Reply:
That property is private and named differently with a "private" word in it. I don't want to do rename refactoring across the board now, and I need to urgently unblock failing tests now. Going to commit as it is, and will file a bug to address minor thing later.